src/Forms/FormField.php

```diff
@@ -91,9 +91,9 @@ class FormField
      * @var null|array
      */
     protected $dataAttributes = [];
-
     /**
      * @var string
+
```

Comment:
This random empty line will break docblock processing

Reply:
Removed!